configuration.ts

```diff
@@ -17,7 +17,7 @@ import { assertParamExists, isWellFormedUlidString, isWellFormedUriString } from
 import { TelemetryConfig, TelemetryConfiguration } from "./telemetry/configuration";
 
 // default maximum number of retry
-const DEFAULT_MAX_RETRY = 15;
+const DEFAULT_MAX_RETRY = 3;
```

Comment:
We should make sure to call out the change to default retries in the changelog too (can do that in the PR that handles bumping the version)